test/commands/utils.ts

```diff
@@ -5,7 +5,7 @@ export function omitHDFChangingFields(
   input: Partial<ExecJSON.Execution> & {profiles: ExecJSON.Profile[]},
 ) {
   return {
-    ..._.omit(input, ['version', 'platform.release', 'profiles[0].sha256']),
+    ..._.omit(input, ['version', 'platform.release', 'profiles[0].sha256', 'profiles[0].version']),
```

Comment:
This change/addition of omission is to account for the version number being applied to the parent profile in the ChecklistMapper. We are using the Heimdall Version currently. The parent profile version is populated from the inspec.yml of the Inspec Profile. Since Heimdall is creating the Parent Profile, the version will get updated and tests will fail.